synchronization/src/test/java/de/cyface/synchronization/MeasurementSerializerTest.java

```diff
@@ -217,10 +283,12 @@ public void testDeserializeMeasurement() throws CursorIsNullException, IOExcepti
      *
      */
     @Test
-    public void testSerializeCompressedMeasurement() throws CursorIsNullException, IOException {
+    public void testSerializeCompressedMeasurement() throws CursorIsNullException {
 
-        final long SERIALIZED_COMPRESSED_SIZE = 24L; // When compression Deflater(level 9, true)
-        final File compressedTransferBytes = oocut.writeSerializedCompressed(loader, 0, persistence);
+        // If you need to change the sample point counts (and this) make sure the test work with the previous counts
+        final long SERIALIZED_COMPRESSED_SIZE = 43L; // When compression Deflater(level 9, true)
```

Comment:
Before changing this number I checked that the new code still ends up this 24 bytes compressed data when using 3/3/3 sensor points. Then I changed the numbers to 21/20/10 and hard-coded the new compressed byte count into this test here.